.circleci/publish_jar.sh

```diff
@@ -6,6 +6,8 @@ JAVA_API_DIR=api/core/java
 SCALA_API_DIR=api/core/scala
 RESOURCE_DIR=$JAR_BUILD_DIR/src/main/resources/resources/djinni_native_libs
 
+bash tools/generate_interfaces.sh
```

Comment:
I think it would be preferable to keep scripts simple (KISS). What do you think of removing the `generate_interfaces.sh` for here and add the corresponding line in `config.yml` as another step ?

Reply:
done